### DISCUSSION TITLE
Production ready?

### DISCUSSION THREAD

**Discussion by denisbredikhin** [CoreWCF/CoreWCF#462] [General]
Hello everyone.
Can someone please tell me if CoreWCF is ready to be used in production?
We are working on migration of big enterprise-level application from to .NET Core and we want to replace WCF with CoreWCF.
We don't mind to fix some issues if the appear and contribute to CoreWCF, but we are not ready to face issues everywhere.

**Comment by MaceWindu** [comment]
It depends. Check list of mising features here https://github.com/CoreWCF/CoreWCF/issues/234

E.g. we need WSDL generation, which is missing here, so we are looking at https://github.com/DigDes/SoapCore instead

**Reply by sharpninja** [reply]
No WSDL?  This is monumentally important.  Wouldn't you need to put you definitions in a share project and the create services for both Framework and Core to enable all of the Visual Studio features that makes API dev so fast with WCF?  And it would also mean that your app would be debugging against Framework when you have both server and client in same solution.

**Reply by birojnayak** [reply]
https://github.com/CoreWCF/CoreWCF/issues/234#issuecomment-703430409 => we are working and should be available soon...

**Reply by sharpninja** [reply]
> [#234 (comment)](https://github.com/CoreWCF/CoreWCF/issues/234#issuecomment-703430409) => we are working and should be available soon...

That is very good news!

**Comment by birojnayak** [comment]
@denisbredikhin few of our contributors are already running apps in Prod using CoreWCF. Please check the compatibility(feature presently supported) and what's upcoming as told by @MaceWindu .... also as we are OS community, we encourage contribution, for that you can start with goodfirstissue and get involved for bigger features... also while  trying out, if you are facing issue, feel free to create issue or discussion and community can provide some guidance..
